src/components/header/header.spec.js

```diff
@@ -53,6 +53,15 @@ describe("Components/Header", () => {
       expect(logo[0]).not.equal(undefined);
     });
 
+    it("should have an anchor tag wrapping the logo", () => {
+      const anchor = header.shadowRoot.querySelector("header a");
+      const logo = anchor.querySelector(".logo-bar svg");
+
+      expect(anchor).to.not.equal(undefined);
+      expect(anchor.getAttribute("href")).to.equal("/");
+      expect(logo).to.not.equal(undefined);
```

Comment:
```js
expect(logo).to.not.equal(undefined);
```

We already test for the logo / `<svg>` above, so I think instead of this line, let's test for a propert `title` attribute on the link instead (which would be good for a11y), e.g.
```js
expect(anchor.getAttribute("title")).to.equal("Greenwood home page");
```

Reply:
@thescientist13 Updated the files as per the suggestions.Thanks!